lib/HTTP/Action.js

```diff
@@ -10,12 +10,12 @@ class HTTPAction extends Action {
     this.httpConfig = httpConfig;
   }
 
-  async request(q, method, headers, body) {
+  async request(q, method = 'POST', headers, body) {
```

Comment:
Then again body seems to mandatory here, according to how it's used. Should the function allow for bodyless request?